### DISCUSSION TITLE
Can change the namespace of the project at same time ?

### DISCUSSION THREAD

**Discussion by flagshang** [ModernRonin/ProjectRenamer#36] [Q&A] [answered]
Can change the namespace of the project at same time ?  thank you.

**Comment by ModernRonin** [comment, marked as answer]
This is a non-trivial change because you need to fix all references, too. ReSharper can do this for you (and they've got the code for doing this since like 10 years, so it's mature.) You just right-click the project, select "Refactor...", then "Adjust namespaces...".

**Reply by flagshang** [reply]
Got it. Thank you.
